docs/docs/keymaps/behaviors/bluetooth.md

```diff
@@ -46,6 +46,9 @@ Here is a table describing the command for each define:
 | `BT_PRV`     | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                                                     |
 | `BT_SEL`     | Select the 0-indexed profile by number; must include a number as an argument in the keymap to work correctly, e.g. `BT_SEL 0`.                                                     |
 | `BT_DISC`    | Disconnect from the 0-indexed profile by number, if it's currently connected and inactive; must include a number as an argument in the keymap to work correctly, e.g. `BT_DISC 0`. |
+| `ADV_OFF`    | Disable advertising and disconnect from all profiles                                                                                                                               |
+| `ADV_ON`     | Enable advertising and attempt to reconnect to profiles                                                                                                                            |
+| `ADV_TOG`    | Toggle advertising on and off                                                                                                                                                      |
 
```

Comment:
I think it might be worth adding a small note here, saying that advertising is on by default.

I see this pattern in some users where they assume they need to set a setting for something to work, just because they see it exists.